x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_errors.ts

```diff
@@ -38,20 +38,20 @@ export function validateBaseProperties(
     );
   }
   if (alertObject.schedule.interval.length < 2) {
-    errors.interval.push(
+    errors['schedule.interval'].push(
       i18n.translate('xpack.triggersActionsUI.sections.alertForm.error.requiredIntervalText', {
         defaultMessage: 'Check interval is required.',
       })
     );
-  } else if (serverRuleType?.minimumScheduleInterval) {
+  } else if (config.minimumScheduleInterval) {
     const duration = parseDuration(alertObject.schedule.interval);
-    const minimumDuration = parseDuration(serverRuleType.minimumScheduleInterval);
+    const minimumDuration = parseDuration(config.minimumScheduleInterval);
     if (duration < minimumDuration) {
-      errors.interval.push(
+      errors['schedule.interval'].push(
         i18n.translate('xpack.triggersActionsUI.sections.alertForm.error.belowMinimumText', {
-          defaultMessage: 'Interval is below minimum ({minimum}) for this rule type',
+          defaultMessage: 'Interval is below minimum ({minimum}).',
```

Comment:
> As a user it isn't entirely clear to me where this minimum has come from - should we provide more details? Perhaps a link to docs? cc @elastic/mlr-docs

If I'm understanding correctly, this minimum comes from the `xpack.alerting.minimumScheduleInterval` setting, therefore could we add something like this?:

```suggestion
          defaultMessage: 'Interval must be greater than the xpack.alerting.minimumScheduleInterval setting value ({minimum}).',
```

Reply:
This seems long? 
<img width="603" alt="Screen Shot 2022-02-22 at 1 12 13 PM" src="https://user-images.githubusercontent.com/13104637/155193427-ed2f48ab-194e-4128-accd-0e07d7dd61c6.png">

Reply:
I like your alternatives in https://github.com/elastic/kibana/pull/125396#issuecomment-1048079770